example/src/index.js

```diff
@@ -113,6 +113,16 @@ class App extends React.Component {
           />
         </div>
 
+        <div className="example">
+          <h3>Custom allowed decimal separators with decimal precision</h3>
+          <NumberFormat
+            thousandSeparator={' '}
+            allowedDecimalSeparators={['.', ',']}
+            decimalSeparator='.'
+            decimalScale={2}
+          />
```

Comment:
Let's keep this example simple by just adding it like this
```
 <NumberFormat
            value={11.11}
            allowedDecimalSeparators={['.', ',']}
            decimalSeparator='.'
            decimalScale={2}
          />
```

Because pasting may not be immediately evident so it's a good idea to have some value and let Devs'/Users' play with it